modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -147,44 +417,84 @@ import {Pagination} from '@workday/canvas-kit-react/pagination';
 
 ### Core
 
-#### Rename Core to Tokens
+#### Remove Labs Core
 
-The distinction between our core and common packages is often unclear and creates confusion around
-what should be imported from where. To help alleviate this and better align with our design
-taxonomy, we've renamed our `core` modules in both Main and Labs to `tokens`. We've also removed the
-`type` default export in the Labs core package in favor of named imports. These changes are listed
-below, all of which are handled by the v5 codemod.
+The Labs `core` package has been removed. The few utilities in that package were either promoted,
+deprecated, or found a better home in another package. These changes are listed below, most of which
+are handled by the v5 codemod.
 
 ##### Automatic Updates
 
-- 🤖 Rename Main `core` import statements to `tokens`
+- 🤖 Move `StaticStates` component to Main `common`
 
-  ```tsx
-  // v4
-  import {colors} from '@workday/canvas-kit-react-core';
+We use `StaticStates` internally for our visual regression tests. It didn't really make sense to
+live in `core`, and it's stable enough to move to Main, so it now lives in `common`.
```

Comment:
Here's another section where indenting the nested content in the list items will improve readability.